packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <>
+      <div className="flex justify-center content-center pt-6">
+        <p className="mt-3 base-300 text-sm font-mono font-semibold"></p>
+        <div className="card w-80 bg-base-100 shadow-xl">
+          <figure className="px-10 pt-10">
+            <Image src={"/avatars/avatar-yan.png"} width={200} height={200} alt="yan's profile avatar on buildguild" />

```

Comment:
You can remove the braces `{}` here

Reply:
Yes. I'm using windows. I didn't know that. I just changed it on my github. I also removed the braces from the image tag. Thanks for the review.

Reply:
Great!

I'm still getting the CRLF line endings. Maybe you have to configure your git locally, re-save the file, and push?

I fixed it for you here: https://github.com/BuidlGuidl/batch0.buidlguidl.com/pull/13/commits/beb096c43b510d989828c0f60826e146dba1424d

Let's see if your settings are working on the next PR.

Merging this, thank you!!

Reply:
Thank you for the kindness. 

I configured it locally and cloned the repository again, I think everything is ok now.